extensions/grapher/src/com/google/inject/grapher/AbstractInjectorGrapher.java

```diff
@@ -38,6 +39,8 @@
   private final AliasCreator aliasCreator;
   private final NodeCreator nodeCreator;
   private final EdgeCreator edgeCreator;
+  private final SubgraphCreator  subCreator;
```

Comment:
could you rename this variable to 'subgraphCreator' (and same below: foundSubs -> foundSubgraphs)?  also, rm the extra space (there's two right now) btw the type & the var name.